packages/store/src/patterns/patternMatchers.js

```diff
@@ -36,10 +37,80 @@ import {
 
 const { quote: q, details: X } = assert;
 const { entries, values } = Object;
+const { ownKeys } = Reflect;
 
 /** @type WeakSet<Pattern> */
 const patternMemo = new WeakSet();
 
+// /////////////////////// Match Helpers Helpers /////////////////////////////
+
+/**
+ * exported primarily for testing.
+ */
+export const defaultLimits = harden({
```

Comment:
Even > 2**332 ;)

I didn't actually have crypto in mind. Instead of explaining individual choices, I'm adding a comment to `defaultLimits` 

```js
/**
 * The actual default values here are, at the present time, fairly
 * arbitrary choices and may change before they settle down. Of course
 * at some point we'll need to stop changing them. But we should first
 * see how our system holds up with these choices. The main criteria
 * is that they be big enough that "normal" innocent programs rarely
 * encounter these limits.
```

I also agree with your observation below that these limits are not intended to cover cryptograhic values. These should have shapes of their own. So resolving.